buildSrc/src/main/kotlin/pklKotlinLibrary.gradle.kts

```diff
@@ -14,13 +14,16 @@
  * limitations under the License.
  */
 import org.gradle.accessors.dm.LibrariesForLibs
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.tasks.KotlinJvmCompile
 
 plugins {
   id("pklJavaLibrary")
-
+  id("org.jetbrains.kotlinx.binary-compatibility-validator")
```

Comment:
I've added Kotlin ABI checking so that the Kotlin version change and JDK 21 change can be tested (w.r.t. library use) before release.